spec/requests/conversion_hosts_spec.rb

```diff
@@ -39,16 +43,18 @@
 
     it "forbids access to a conversion host resource without an appropriate role" do
       api_basic_authorize
-      conversion_host = FactoryBot.create(:conversion_host, :resource => FactoryBot.create(:vm))
+      conversion_host = FactoryBot.create(:conversion_host, :resource => FactoryBot.create(:vm_openstack))
       get(api_conversion_host_url(nil, conversion_host))
 
       expect(response).to have_http_status(:forbidden)
     end
   end
 
   context "create" do
-    let(:vm) { FactoryBot.create(:vm) }
-    let(:host) { FactoryBot.create(:host) }
+    let(:zone) { FactoryBot.create(:zone, :name => "api_zone") }
```

Comment:
`name` shouldn't be necessary since it is sequenced

Reply:
Updated.